alerting/src/main/kotlin/org/opensearch/alerting/InputService.kt

```diff
@@ -105,6 +122,28 @@ class InputService(
         }
     }
 
+    private fun updateInputQueryWithFindingDocIds(
```

Comment:
this should be a common methods used in all the monitor types

Reply:
I thought the same - but then I saw that the search query is executed on a different way depending of the monitor type.
Ie. [here](https://github.com/opensearch-project/alerting/blob/main/alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt#L456-L521) you can see how the doc level monitor is getting the matching docs. So, for example, doc level monitor iterates through the list of indices and getting the document by document. That's why I adjusted getting the matched docIds on doc level monitor to be aligned with existing logic. Check it out [here](https://github.com/stevanbz/alerting/pull/1/files#diff-68866b22ed9703814b4d5db8d3488872bcb972086ecaca10c9b8bfd54db981bcR508-R511)